### DISCUSSION TITLE
什么设置公网ip

### DISCUSSION THREAD

**Discussion by deep-learner-ly** [halo-dev/halo#5108] [Q&A]
### System information

curl -i http://localhost:8090/
有响应
但将localhost换成公网ip
无响应

### What is the project operation method?

Docker

### What happened?

curl -i http://localhost:8090/
有响应
但将localhost换成公网ip
无响应
请教怎么解决

**Comment by ruibaby** [comment]
有将 8090 端口在公网开放吗？

这似乎不属于 issue，我将转移到 discussion。
